R/update-input.R

```diff
@@ -667,22 +692,47 @@ selectizeJSON <- function(data, req) {
   # convert a single vector to a data frame so it returns {label: , value: }
   # later in JSON; other objects return arbitrary JSON {x: , y: , foo: , ...}
   data <- if (is.atomic(data)) {
```

Comment:
I think the whole `data <-` block can be removed, since `data` should already be a data frame at this point, right?

Reply:
oops, somehow missed it during merge of upstream